sunpy/physics/differential_rotation.py

```diff
@@ -173,7 +175,8 @@ def rot_hpc(x, y, tstart, tend, frame_time='synodic', rot_type='howard', **kwarg
                                          y.to(u.arcsec).value,
                                          b0_deg=vstart["b0"].to(u.deg).value,
                                          l0_deg=vstart["l0"].to(u.deg).value,
-                                         dsun_meters=(constants.au * sun.sunearth_distance(t=dstart)).value,
+                                         dsun_meters=(constants.au *
```

Comment:
Don't we have a function that can directly take a coordinate object as input, and interchange within frames?